implementations/rust/ockam/ockam/src/credential.rs

```diff
@@ -8,8 +8,10 @@ mod exchange;
 #[cfg(feature = "std")]
 pub use exchange::*;
 
+use core::fmt;
 use core::marker::PhantomData;
 use core::time::Duration;
+use data_encoding::BASE32_DNSSEC;
```

Comment:
I don't think we need base32 in out codebase. The decision was to use hex everywhere for the time being

Reply:
@twittner is data_encoding better than hex crate we already use?

Reply:
I was hoping we could agree to use a more compact encoding, but for the time being I removed data-encoding and use hex as requested.

Reply:
@twittner I think we're open to suggestions, but to be clear: making a change and expecting reviewer to notice it, is not the way to bring on a discussion